bash/solutii/Alex_Tetelea/compare.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+function rec {
```

Comment:
In bash, este recomandat sa scrii functii in momentul cand le vei apela de macar doua ori.